README.md

```diff
@@ -52,26 +52,33 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 ![CoatRack architecture overview](./spring-boot/admin/src/main/resources/static/images/coatrack-architecture-overview.png)
 
-## Install
+## Installation for local usage
```

Comment:
"local usage" seems misleading, because the goal of this approach is that it can be used anywhere. 

So I would just change back to "Install". 

The previous text under "install" was more like a placeholder, as it was not about installation at all. That stuff should be moved to another page or another subsection.

Reply:
Done. I changes the title to 'Install' again and moved the development part to the CoatRack wiki.